packages/rpms/utils/helper_function.sh

```diff
@@ -86,8 +88,14 @@ build_package(){
 }
 
 get_checksum(){
+    if [[ "${RELEASE_PACKAGE}" == "yes" ]]; then
```

Comment:
This should also prompt some changes when we call *rpmbuild* [link](https://github.com/wazuh/wazuh/pull/22435/commits/2670d65cbc4ff6ede6a1e512d1f565f310b6e943#diff-9d723f00fd3a519168b1b75d70d5f05abb9174e4f344d66a3d5b49b6042f38caR83). I will make these changes on my branch.